### PR TITLE
[W6.1e][W14-B2]Sun Bangjie

### PR DESCRIPTION
**Pull request by SunBangjie**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,6 +36,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsHasAnyNull() throws Exception {
+
+        // empty list
+        assertHasNoNull();
```
Comment (SunBangjie) [inline test/java/seedu/addressbook/common/UtilsTest.java:43]: isAnyNull_noNull_returnsFalse()

Comment (okkhoy) [reply test/java/seedu/addressbook/common/UtilsTest.java:43]: This should be used for the test name; not individual asserts!

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -36,6 +36,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsHasAnyNull() throws Exception {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:40]: This is a correctly written test. 
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.

**Comment by okkhoy** [comment]
@SunBangjie 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**